libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Store/SFSDKEventStoreManager.m

```diff
@@ -75,13 +75,14 @@ - (void) storeEvent:(SFSDKInstrumentationEvent *) event {
 
     // Copies event, to isolate data for I/O.
     SFSDKInstrumentationEvent *eventCopy = [event copy];
-    if (!eventCopy || ![eventCopy jsonRepresentation]) {
+    NSData *eventJsonData = [eventCopy jsonRepresentation];
+    if (!eventCopy || !eventJsonData) {
         return;
     }
     if (![self shouldStoreEvent]) {
         return;
     }
-    NSData *encryptedData = self.dataEncryptorBlock([eventCopy jsonRepresentation]);
```

Comment:
`[eventCopy jsonRepresentation]` was being called twice within this method, so moved it to a local variable `eventJsonData`.